### ISSUE TITLE
Implementation: Frontend for Creating Community / Getting Community

### ISSUE THREAD

**Issue by jcbod** [bounswe/2021SpringGroup3#133]
Implementing the view controllers for the following controllers:
* CreateCommunityController
* GetCommunityController

**Deadline**: 08.06.2021

**Comment by orhungorkem** [comment]
* I implemented GetCommunityViewController by adding a form connection where the form is an input space for searched community name.   
* Then I coded the function to link the community listing template. When the search input is taken, program redirects to result page. 
* I also added a "see posts" button to get posts of the searched community. (Assuming GetPostsViewController is implemented.)

**Comment by irem-zeynep** [comment]
- I implemented a view controller named **GettingCommunitiesByPublicityViewController** for getting communities by their publicities.  
- Created templates for viewing and searching communities with their publicity. One can select public/private from the dropdown and see communities based on the selection. 
- I implemented the function to create a community in the **CreateCommunityViewController**.  This function acts as a bridge between our REST API and creates a community template.
- I created/updated templates for creating community, searching community with name/publicity, viewing communities.

While writing templates, I used _Bootstrap_  and _AgGrid_  libraries.
